packages/devtools_app/lib/src/flutter/console.dart

```diff
@@ -184,11 +152,11 @@ class DeleteControl extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ConsoleControl(
+    return ToolbarAction(
       icon: Icons.delete,
-      tooltip: tooltip,
       onPressed: onPressed,
-      buttonKey: buttonKey,
+      key: buttonKey,
```

Comment:
Doesn't this change break tests?

Reply:
I updated the related tests for the slightly changed expectations.